ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -61,6 +62,82 @@ export const styles = reactStyles({
   },
 });
 
+const WelcomeHeader = () => {
+  return <FlexColumn style={{marginLeft: '3%', width: '50%'}}>
+    <FlexRow>
+      <FlexColumn>
+        <Header style={{fontWeight: 500, color: colors.secondary, fontSize: '0.92rem'}}>
+          Welcome to the</Header>
+        <Header style={{textTransform: 'uppercase', marginTop: '0.2rem'}}>
+          Researcher Workbench</Header>
+      </FlexColumn>
+      <FlexRow style={{alignItems: 'flex-end', marginLeft: '1rem'}}>
+        <img style={styles.welcomeMessageIcon} src='/assets/images/workspace-icon.svg'/>
+        <img style={styles.welcomeMessageIcon} src='/assets/images/cohort-icon.svg'/>
+        <img style={styles.welcomeMessageIcon} src='/assets/images/analysis-icon.svg'/>
+      </FlexRow>
+    </FlexRow>
+    <SmallHeader style={{color: colors.primary, marginTop: '0.25rem'}}>
+      The secure platform to analyze <i>All of Us</i> data</SmallHeader>
```

Comment:
nit: can use the `<AoU>` component here

Reply:
and other places https://github.com/all-of-us/workbench/pull/5373